### PR TITLE
feat: eliminated duplicate init call need for non-react apps

### PR DESCRIPTION
**Pull request by alisher-aituarov**
Removed duplicate supertokens init calls for angular|vue boilerplate apps

### CODE REVIEW

File: boilerplate/frontend/angular-prebuilt/README.md [hunk 1]
```diff
@@ -51,9 +51,9 @@ The app will start on `http://localhost:4200`
 
 -   The frontend code is located in the `/src` folder.
 -   Our frontend will have 3 Angular components, a `home`, `auth` and `root` component.
--   The `home` component will use `supertokens-web-js` SDK for session management and the `auth` component will use `supertokens-auth-react` for Authentication flows and UI.
--   Since both the `home` and `auth` components (since `supertokens-auth-react` SDK uses `supertokens-web-js`) depend on `supertokens-web-js`, we can initialize this library in our root component. This will also reduce the bundle sizes for the `home` and `auth` component.
--   The `root` component contains our routing logic with refrences to the the `home` and `auth` component modules to enable code splitting.
+-   `supertokens-auth-react` will be used in `home` for session management and in the `auth` for Authentication flows and UI.
+-   Since `supertokens-auth-react` SDK uses `supertokens-web-js` underneath we only need to initialize `supertokens-auth-react` on the main route.
```
Comment (porcellus) [inline boilerplate/frontend/angular-prebuilt/README.md:55]: I'm not sure if this line needs to be here anymore.

**Review by porcellus — CHANGES_REQUESTED** [review]

**Review by porcellus — APPROVED** [review]

File: boilerplate/frontend/angular-prebuilt/README.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ This demo app demonstrates the following use cases:
 -   Forgot password flow
```
Comment (rishabhpoddar) [inline boilerplate/frontend/angular-prebuilt/README.md:9]: the boilerplates need to be updated to use the new way of adding the pre built UI

**Review by rishabhpoddar — CHANGES_REQUESTED** [review]

File: lib/ts/utils.ts [hunk 3]
```diff
@@ -493,50 +490,46 @@ async function setupFullstack(answers: Answers, folderName: string, userArgument
...
-    fs.rmSync(`${folderName}/${normaliseLocationPath(selectedFullStack.location.config.backend.configFiles)}`, {
-        recursive: true,
-        force: true,
-    });
+        // Remove the configs folder
+        fs.rmSync(`${folderName}/${normaliseLocationPath(config.configFiles)}`, {
+            recursive: true,
+            force: true,
+        });
+    }
```
Comment (rishabhpoddar) [inline lib/ts/utils.ts:533]: requires version change

**Review by nkshah2 — APPROVED** [review]

File: boilerplate/fullstack/next/pages/auth/[[...path]].tsx [hunk 4]
```diff
@@ -3,15 +3,17 @@ import React, { useEffect } from "react";
 import styles from "../../styles/Home.module.css";
 import dynamic from "next/dynamic";
 import SuperTokens from "supertokens-auth-react";
+import { canHandleRoute, getRoutingComponent } from "supertokens-auth-react/ui";
+import { PreBuiltUIList } from "../../config/frontendConfig";
 
 const SuperTokensComponentNoSSR = dynamic<React.ComponentProps<typeof SuperTokens.getRoutingComponent>>(
```
Comment (rishabhpoddar) [inline boilerplate/fullstack/next/pages/auth/[[...path]].tsx:9]:
```suggestion
const SuperTokensComponentNoSSR = dynamic<{}>(
```

**Review by rishabhpoddar — CHANGES_REQUESTED** [review]

**Review by rishabhpoddar — APPROVED** [review]

**Pull request merged**